data/ecosystems/a/astar-network.toml

```diff
@@ -528,9 +562,24 @@ url = "https://github.com/Phala-Network/subbridge.js"
 [[repo]]
 url = "https://github.com/PlasmNetwork/defillama-app"
 
+[[repo]]
+url = "https://github.com/rairtech/solidity"
+
 [[repo]]
 url = "https://github.com/realGhostChain/ERC721Envious"
 
+[[repo]]
+url = "https://github.com/RottenKiwi/Panorama-Swap-INK-SC"
+
+[[repo]]
+url = "https://github.com/SfyLabs-HASC/Age-of-Chronos-Rebellion-s-Echo"
+
+[[repo]]
+url = "https://github.com/SkylabCorporation/SkylabCorp"
+
+[[repo]]
+url = "https://github.com/stated-tech/Counter.ink"
```

Comment:
how's this repo using Astar? eg, can you point out where in the code I can find traces of Astar usage?